lib/active_model/serializer/reflection.rb

```diff
@@ -42,17 +42,17 @@ def initialize(*)
 
       def link(name, value = nil, &block)
         @_links[name] = block || value
-        nil
+        :nil
```

Comment:
Yes. For more details, there are two use cases to consider for the relationship block:
* Where the block return the value we want to use for the relationship:
```ruby
has_one :author do
  meta some: :thing
  object.own_author
end
```
* Where the block return some value to define that we want to use the default relationship:
```ruby
has_one :author do
  meta some: :thing
end
```

For the latest, I defined the value to be `:nil` but it could be change to anything else if a better suggestion pops up since this is the internal API.